onnx/backend/test/case/node/ai_onnx_ml/binarizer.py

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+import onnx
+
+from ...base import Base
+from .. import expect
+
+
+def compute_binarizer(x, threshold=None):
```

Comment:
Is it possible to call the reference-implementation (in `onnx/reference/ops/aionnxml`)?

Reply:
I was thinking that it is better to put reference implementation as close to the sample as possible. Now that noth @xadupre and @gramalingam commented the same. I must be wrong for this. I will move the implementation back to where it is - the reference implementation location.